packages/theme/src/cli/utilities/theme-environment/proxy.ts

```diff
@@ -28,21 +29,25 @@ const IGNORED_ENDPOINTS = [
   '/mini-profiler-resources',
   '/web-pixels-manager',
   '/wpm',
+  '/services/',
```

Comment:
There's a request to `/services/...` that overflows parsing headers (https://github.com/Shopify/develop-advanced-edits/issues/324). This request is related to Shop Pay so I've added this prefix to the ignored requests that we don't proxy.